package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "movieland",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
For improved package management, I propose moving several dependencies to the dev dependencies. This update separates testing and build-related packages from production dependencies by moving them to dev dependencies. These modifications aid in streamlining the production environment by removing redundant development dependencies.